front/src/modules/activities/components/TaskGroups.tsx

```diff
@@ -1,9 +1,65 @@
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { IconCheckbox } from '@tabler/icons-react';
+
+import { Button, ButtonVariant } from '@/ui/button/components/Button';
+import { ActivityType } from '~/generated/graphql';
+
+import { useOpenCreateActivityDrawer } from '../hooks/useOpenCreateActivityDrawer';
 import { useTasks } from '../hooks/useTasks';
 
 import { TaskList } from './TaskList';
 
+const StyledTaskGroupEmptyContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  flex: 1 0 0;
+  flex-direction: column;
+  gap: 8px;
+  justify-content: center;
+  padding-bottom: ${({ theme }) => theme.spacing(16)};
```

Comment:
I had to separate padding because `padding` alone with multiple spacing was not well rendered with prettier